app/helpers/application_helper.rb

```diff
@@ -69,4 +69,16 @@ def render_flash_stream
     turbo_stream.update "flash", partial: "layouts/notification"
   end
 
+  def vehicle_reports(car)
+    if @vehicle_reports = VehicleReport.where(reservation_id: car.reservations.ids).order(:updated_at).present?
```

Comment:
Another example of using tell don't ask. Will definitely benefit us going forward